source/extensions/filters/http/aws_lambda/config.h

```diff
@@ -56,6 +56,7 @@ class AWSLambdaProtocolExtensionConfig
     return session_token_;
   }
   const absl::optional<std::string> &roleArn() const { return role_arn_; }
+  const absl::optional<bool> &disableRoleChaining() const { return disable_role_chaining_; }
```

Comment:
Why is this optional, can't it just be bool? It's either true or false

Reply:
I like the not setting in case we changed it in the future but I think that its actually quite safe to be a bool, I shall change it